dashboard/app/assets/javascripts/videos.js.erb

```diff
@@ -29,6 +29,12 @@ function showVideoDialog(options, forceShowVideo) {
   }
 
   if (dashboard.clientState.hasSeenVideo(options.key) && forceShowVideo === false) {
+    // Anything we were going to do when the video closed, we ought to do
```

Comment:
As far as I can tell, although it may have only been noticeable in this case where we have a post-level video at the very end of a script, and we're expecting to redirect to a certificate.

If you think this behavior was intentional before, I'll be happy to investigate further.  I'd be surprised if it was though - it made the "Next Level" button in the feedback dialog a no-op, which felt bad enough I think somebody would have complained about it by now.